Brokerages/app.config

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
```

Comment:
This is a new file, and since the brokerages project is a library and not an exe, I think this file is not necessary

Reply:
These binding redirects were added automatically by VS 2017 when I updated the NuGet reference to the new version of IKVM.
But you're right, they are not needed at all -- thankfully we don't use multiple versions of IKVM in LEAN 😄 
I will remove them.